tslint.yaml

```diff
@@ -99,9 +99,6 @@ rules:
         options:
             - spaces
             - 4
-    max-line-length:
-        options:
-            - 120
```

Comment:
Was this needed for this PR?

Reply:
No, but it was annoying that it was making it not compile before I committed the code and the git hook ran